src/gui/properties/peerlistwidget.cpp

```diff
@@ -138,6 +136,67 @@ PeerListWidget::~PeerListWidget()
     delete m_copyHotkey;
 }
 
+void PeerListWidget::displayToggleColumnsMenu(const QPoint&)
+{
+    QMenu hideshowColumn(this);
+    hideshowColumn.setTitle(tr("Column visibility"));
+    QList<QAction*> actions;
+    for (int i = 0; i < PeerListDelegate::IP_HIDDEN; ++i) {
```

Comment:
It says `resolvePeerCountries()` needs an object but it compiles with
`!Preferences::instance()->resolvePeerCountries()`
@sledgehammer999 i don't really know what this does and i doubt i'd understand even if you told me. I'll probably make a mistake again and unless it's critical maybe you should do it?